Tensile/KernelWriterAssembly.py

```diff
@@ -4191,6 +4249,9 @@ def graFinalOffsets(self, kernel, tP):
                              "compute offset diff (tileDim)")
               else:
                 tileStride   = kernel[tP["lsp"]] * (perp*tVW + sPara*tVS)
+                #TODO remove wave_offset start for global fetching  TransposeLDS
```

Comment:
Does this TODO need to be fixed before this checkin?

Reply:
lets keep the TODO for now